config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -0,0 +1,81 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-storage-kind-disruptive
+    always_run: false
+    optional: true
+    # All the files owned by sig-storage. Keep in sync across
+    # all sig-storage-jobs
+    #
+    # pkg/controller/volume
+    # pkg/kubelet/volumemanager
+    # pkg/volume
+    # pkg/util/mount
+    # test/e2e/storage
+    # test/e2e/testing-manifests/storage-csi
+    run_if_changed: '^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)'
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    annotations:
+      testgrid-create-test-group: 'true'
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20230421-ec4335b54b-1.27
+        command:
+        - wrapper.sh
+        args:
+        - /bin/sh
+        - -xc
+        - >
+          make WHAT="github.com/onsi/ginkgo/v2/ginkgo k8s.io/kubernetes/test/e2e/e2e.test k8s.io/kubernetes/cmd/kubectl" &&
+          curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" kind &&
+          kind build node-image --image node:latest &&
+          kind create cluster --image node:latest &&
+          trap 'kind export logs "${ARTIFACTS}/kind"' EXIT &&
+          KUBERNETES_PROVIDER=local KUBECONFIG=${HOME}/.kube/config GINKGO_PARALLEL_NODES=1 E2E_REPORT_DIR=${ARTIFACTS} hack/ginkgo-e2e.sh -ginkgo.focus="\[Feature:Kind\].*\[Disruptive\]"
```

Comment:
This doesn't focus on SIG Storage tests?

Reply:
Right now, there is only one test with `Feature:Kind`, but I added `[sig-storage]` to focus.